src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -22,62 +22,101 @@
 import java.util.stream.Collectors;
 
 public class Statistics {
-  private static final double LOWER_END_VAL = -1.0;
-  private static final double UPPER_END_VAL = 1.0;
-  private static final BigDecimal INTERVAL = BigDecimal.valueOf(0.2);
-  private static final BigDecimal UPPER_END = BigDecimal.valueOf(UPPER_END_VAL);
-  private static final BigDecimal LOWER_END = BigDecimal.valueOf(LOWER_END_VAL);
+  private static final double LOWER_SCORE_VAL = -1.0;
+  private static final double UPPER_SCORE_VAL = 1.0;
+  private static final double LOWER_MAGNITUDE_VAL = 0.0;
+  private static final double UPPER_MAGNITUDE_VAL = 2.0;
+  private static final double SCORE_INTERVAL_VAL = 0.2;
+  private static final double MAGNITUDE_INTERVAL_VAL = 0.3;
+  private static final BigDecimal SCORE_INTERVAL = BigDecimal.valueOf(SCORE_INTERVAL_VAL);
+  private static final BigDecimal UPPER_SCORE = BigDecimal.valueOf(UPPER_SCORE_VAL);
+  private static final BigDecimal LOWER_SCORE = BigDecimal.valueOf(LOWER_SCORE_VAL);
+  private static final BigDecimal MAGNITUDE_INTERVAL = BigDecimal.valueOf(MAGNITUDE_INTERVAL_VAL);
+  private static final BigDecimal LOWER_MAGNITUDE = BigDecimal.valueOf(LOWER_MAGNITUDE_VAL);
+  private static final BigDecimal UPPER_MAGNITUDE = BigDecimal.valueOf(UPPER_MAGNITUDE_VAL);
 
-  // Contains sentiment scores in the range [-1, 1] with given intervals.
-  private Map<Range, Integer> aggregateValues;
+  // Contains sentiment scores and magnitudes with given intervals.
+  private Map<Range, Integer> aggregateScores;
+  private Map<Range, Integer> aggregateMagnitude;
```

Comment:
Can you go into more detail here about what the sentiment scores and magnitude scores actually are?